master_asteroid_list.json

```diff
@@ -0,0 +1,66 @@
+[
+    {
+        "file": "small_asteroid0",
+        "categoryname": "Fragments",
+        "price": "0",
+        "mass": "100",
+        "volume": "1000",
+        "description": "@cargo/gems.image@A chunk of asteroid. Can be mined for ore if further shot upon."
```

Comment:
Curious about the formatting at the start of the description - `@cargo/gems.image@A` - any insight? It is getting parsed and used by something? If so, we might want to extract that to another field. Again, just curious.

Reply:
I suspect this supplies the images that come with the description in the game.